### PR TITLE
[HAS] Move prometheus to staging and production overlays

### PR DESCRIPTION
**Pull request by johnmcollier**
Based on errors in https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/redhat-appstudio_application-service/307/pull-ci-redhat-appstudio-application-service-main-application-service-e2e/1652114871954706432, I'm wondering if having two links to kustomize overlays in https://github.com/redhat-appstudio/application-service is tripping up the tests. Moving the prometheus overlays (which are only needed in staging and production) to confirm.

### CODE REVIEW

**Review by kim-tsao — APPROVED** [review]

File: components/has/development/kustomization.yaml [hunk 1]
```diff
@@ -2,3 +2,4 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 resources:
   - ../base
+  - https://github.com/redhat-appstudio/application-service/config/prometheus/?ref=df4cd6732afa5f13d8d354b493ed11958288ac31
```
Comment (kim-tsao) [inline components/has/development/kustomization.yaml:5]: I think this should be under staging?

Comment (johnmcollier) [reply components/has/development/kustomization.yaml:5]: 🤦

Comment (johnmcollier) [reply components/has/development/kustomization.yaml]: Fixed

**Review by kim-tsao — APPROVED** [review]
/lgtm

**Pull request merged**